PhysicsTools/PatAlgos/python/slimming/applySubstructure_cff.py

```diff
@@ -131,7 +76,10 @@ def applySubstructure( process, postfix="" ) :
         e.toModify(getattr(process,'nb1AK8PuppiSoftDrop'+postfix), cuts = ['pt > 999999', 'pt > 999999', 'pt > 999999'] )
         e.toModify(getattr(process,'nb2AK8PuppiSoftDrop'+postfix), cuts = ['pt > 999999', 'pt > 999999', 'pt > 999999'] )
 
-
+    from RecoJets.JetProducers.nJettinessAdder_cfi import Njettiness
+    addToProcessAndTask('NjettinessAK8Subjets'+postfix, Njettiness.clone(), process, task)
+    getattr(process,"NjettinessAK8Subjets"+postfix).src = cms.InputTag("ak8PFJetsPuppiSoftDrop"+postfix, "SubJets")
+    getattr(process,"NjettinessAK8Subjets"+postfix).cone = cms.double(0.8)
```

Comment:
I'm guessing from the commit comments that simple ```cone = 0.8```
did not work.
May I rephrase my earlier question: where is this parameter actually used?

Reply:
after some digging, I see that there was a "cone" parameter in this module config until  5 years ago. 
The suggestion to drop the type specs correctly pointed to a buggy code.

Reply:
Correct. This should be "R0". I'm working on it now.